extensions/kubeapi/rbac/rbac.go

```diff
@@ -67,7 +67,7 @@ var RoleTemplateGroupVersionResource = schema.GroupVersionResource{
 	Resource: "roletemplates",
 }
 
-// ProjectRoleTemplateBindingGroupVersionResource is the required Group Version Resource for accessing project role template bindings in a cluster, using the dynamic client.
+// ProjectRoleTemplateBindingGroupVersionResource is the required Group Version Resource for accessing projectroletemplatebindings in a cluster, using the dynamic client.
```

Comment:
If I check that right, there's a diff here with this: https://github.com/rancher/shepherd/pull/174/files

Reply:
This function doesn't exist in 2.7 so resolving this